wagtail/admin/static_src/wagtailadmin/js/vendor/tag-it.js

```diff
@@ -316,6 +328,15 @@
         _tags: function() {
             return this.tagList.find('.tagit-choice:not(.removed)');
         },
+        
+        // Debouncer for tags that accepts a delay arg
+        debounce: function(func, delay) {
```

Comment:
We already have a debounce util in https://github.com/wagtail/wagtail/blob/main/client/src/utils/debounce.ts

Reply:
Oh, I don't know how I missed that, thanks!